src/lib/auth-client.ts

```
@@ -0,0 +1,7 @@
+import { createAuthClient } from "better-auth/react";
+
+const authClient = createAuthClient({
+  baseURL: process.env.NEXT_PUBLIC_API_URL,
+});
+
+export const {} = authClient;
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)